src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -1,10 +1,17 @@
 package cromwell.engine
 
+import cromwell.binding.FullyQualifiedName
+import cromwell.engine.ExecutionIndex.ExecutionIndex
 import cromwell.engine.ExecutionStatus.ExecutionStatus
 
 package object db {
   // CallStatus doesn't exist as an enum already, or reuse ExecutionStatus?
   type CallStatus = ExecutionStatus
+  trait DatabaseKey {
+    val fqn: FullyQualifiedName
+    val index: ExecutionIndex
+  }
+  case class ExecutionDatabaseKey(fqn: FullyQualifiedName, index: ExecutionIndex) extends DatabaseKey
```

Comment:
Beyond that, if I'm looking at things correctly at the very least it'd need a much more specific name than `DatabaseKey`